gulpfile.js

```diff
@@ -42,6 +60,82 @@ function createStringSource(filename, content) {
   return source;
 }
 
+function createBundle(defines) {
+  defines = defines || DEFINES;
+
+  var versionJSON = JSON.parse(
+    fs.readFileSync(BUILD_DIR + 'version.json').toString());
+  var bundle = makeFile.bundle;
```

Comment:
With the comment above fixed, isn't this line redundant, since you could just reference the global `bundle`?